ui/pages/mission/[tokenId].tsx

```diff
@@ -416,6 +485,23 @@ export default function MissionProfile({ mission }: ProjectProfileProps) {
                       </Link>
                     </div>
                   )}
+                    {/* Send payouts and tokens Buttons - only shown to managers */}
+                    {account && isManager &&(
+                      <div className="flex flex-col sm:flex-row gap-2 mt-4 w-full sm:w-auto sm:absolute sm:right-0.5 sm:top-[250px]">
+                        <PrivyWeb3Button
+                          requiredChain={DEFAULT_CHAIN_V5}
+                          className="gradient-2 rounded-full noPadding text-sm leading-none flex-1 sm:w-[175px]"
+                          label={<span className="whitespace-nowrap">Send Tokens</span>}
```

Comment:
Just a thought: Do we display how many tokens or how much ETH is available before people send tokens or send payouts? Might be worth adding that as well so people know what they should expect to get for clicking these buttons.

Reply:
Also, we've been talking about where these buttons should live...

Maybe we could create a separate "Tokens" tab where people can view their token balance, release payouts, and reserved tokens?

Having this live in the header I don't think is the best home for these buttons